app/Http/Controllers/ScheduleController.php

```
@@ -3,23 +3,38 @@
      */
-    public function index()
+    public function index(ScheduleFilterRequest $request)
     {
-        $main = Schedule::with("department", "employee")->get();
-        $data = json_decode('{}');
-        $data->message = "Successfully fetch.";
-        $data->success = true;
-        $data->data = $main;
-        return response()->json($data);
+        $validatedData = $request->validated();
+        $startDate = $request->has('start_date') ? Carbon::parse($validatedData['start_date']) : Carbon::now()->subMonth()->startOfMonth()->startOfDay();
```

PHPMD (2.15.0)

Avoid using static access to class '\Carbon\Carbon' in method 'index'. (Clean Code Rules)

(StaticAccess)